plugins/outputs/groundwork/log_adapter.go

```diff
@@ -0,0 +1,87 @@
+package groundwork
+
+import (
+	"context"
+	"encoding/json"
+	"log/slog"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+)
+
+// NewLogger creates telegraf.Logger adapter for slog.Logger
+func NewLogger(l telegraf.Logger) *slog.Logger {
```

Comment:
Do not export the function as this is only used in here...
```suggestion
func newLogger(l telegraf.Logger) *slog.Logger {
```

Reply:
done